Client/Frontend/Settings/ThemeSettings/ThemeSettingsController.swift

```diff
@@ -76,6 +76,7 @@ class ThemeSettingsController: ThemedTableViewController, StoreSubscriber {
     override func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
         if isReduxIntegrationEnabled {
+            store.dispatch(ActiveScreensStateAction.closeScreen(.themeSettings))
```

Comment:
@yoanarios I think we want a matching `closeScreen` action here so I went ahead and added it, but LMK if this may be incorrect. Without this, however, the `activeScreens` just continues growing with multiple copies of `AppScreenState.themeSettings`